src/popup.js

```diff
@@ -519,6 +519,35 @@ window.addEventListener('load', function(evt) {
 
     window.close();
   });
+  
+  $(".wcmmode").click(function(e) {
+    e.preventDefault();
+    
+    var $this = $(this),
+        url = getUrlWithUpdatedQueryString(pageDetails.location, null),
```

Comment:
Without specifying a value for the "key" argument, does this method call return the same value as pageDetails.location.href?